components/molecules/ContributorListTableRow/contributor-list-table-row.tsx

```diff
@@ -179,7 +177,12 @@ const ContributorListTableRow = ({
 
         {/* Column: Last Contribution */}
         <div className={clsx("flex-1 lg:max-w-[130px]  flex text-light-slate-11 justify-center ")}>
-          <div className="flex">{<p>{contributor.author_login ? getLastContributionDate(mergedPrs) : "-"}</p>}</div>
+          <div className="flex flex-col">
+            <p>{contributor.author_login ? getLastContributionDate(mergedPrs) : "-"}</p>{" "}
+            <p className="hidden text-sm font-normal md:inline-flex text-light-slate-9">
+              {truncateString(user?.company || getLastContributedRepo(data), 20)}
```

Comment:
We are incrementally moving out of this approach in truncating string, you should look at https://github.com/open-sauced/app/issues/1763 for the new approach 👍